package.json

```diff
@@ -5,13 +5,13 @@
   "main": "src/server.js",
   "engines": {
     "node": "18.x",
-    "yarn": "1.22.x"
+    "npm": "9.5.x"
```

Comment:
This feels quite restrictive - I've got `npm` version 9.6.7, and the latest is 10.2.5

Perhaps just `9.x` if you're worried about breaking changes? If anyone wants to use 10.x then they can test it and add it to the specification :)

Reply:
At some point we should upgrade to node 20 as well (it's now LTS), but that's a separate change!